src/ImageSharp/Formats/WebP/LosslessUtils.cs

```diff
@@ -0,0 +1,656 @@
+                }
+
+                ++y;
+                if ((y & mask) is 0)
+                {
+                    predRowIdxStart += tilesPerRow;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Reverses the color space transform.
+        /// </summary>
+        /// <param name="m">The color transform element.</param>
+        /// <param name="pixelData">The pixel data to apply the inverse transform on.</param>
+        /// <param name="start">The start index of reverse transform.</param>
+        /// <param name="numPixels">The number of pixels to apply the transform.</param>
+        public static void TransformColorInverse(Vp8LMultipliers m, Span<uint> pixelData, int start, int numPixels)
+        {
+            int end = start + numPixels;
+            for (int i = start; i < end; i++)
+            {
+                uint argb = pixelData[i];
+                sbyte green = (sbyte)(argb >> 8);
+                uint red = argb >> 16;
+                int newRed = (int)(red & 0xff);
+                int newBlue = (int)argb & 0xff;
+                newRed += ColorTransformDelta((sbyte)m.GreenToRed, (sbyte)green);
+                newRed &= 0xff;
+                newBlue += ColorTransformDelta((sbyte)m.GreenToBlue, (sbyte)green);
+                newBlue += ColorTransformDelta((sbyte)m.RedToBlue, (sbyte)newRed);
+                newBlue &= 0xff;
+
+                pixelData[i] = (argb & 0xff00ff00u) | ((uint)newRed << 16) | (uint)newBlue;
+            }
+        }
+
+        /// <summary>
+        /// This will reverse the predictor transform.
+        /// The predictor transform can be used to reduce entropy by exploiting the fact that neighboring pixels are often correlated.
+        /// In the predictor transform, the current pixel value is predicted from the pixels already decoded (in scan-line order) and only the residual value (actual - predicted) is encoded.
+        /// The prediction mode determines the type of prediction to use. The image is divided into squares and all the pixels in a square use same prediction mode.
+        /// </summary>
+        /// <param name="transform">The transform data.</param>
+        /// <param name="pixelData">The pixel data to apply the inverse transform.</param>
+        /// <param name="output">The resulting pixel data with the reversed transformation data.</param>
+        public static void PredictorInverseTransform(Vp8LTransform transform, Span<uint> pixelData, Span<uint> output)
+        {
+            int processedPixels = 0;
+            int yStart = 0;
+            int width = transform.XSize;
+            Span<uint> transformData = transform.Data.GetSpan();
+
+            // First Row follows the L (mode=1) mode.
+            PredictorAdd0(pixelData, processedPixels, 1, output);
+            PredictorAdd1(pixelData, 1, width - 1, output);
+            processedPixels += width;
+            yStart++;
+
+            int y = yStart;
```

Comment:
yStart is only used here and has a constant value of 1, so we can either use int y = 1 here (and remove lines 178 and 186) or use const int yStart = 1 and remove only line 186.